packages/containers/src/List/List.container.js

```diff
@@ -180,8 +231,39 @@ class List extends React.Component {
 				props.toolbar.filter.value = state.searchQuery;
 			}
 
-			props.toolbar.actionBar = { actions: {} };
+			props.toolbar.actionBar = { actions: {}, multiSelectActions: {} };
+
+			// settings up multi selection
+			if (props.multiSelectActions && props.multiSelectionKey) {
+				props.list.itemProps.onToggle = this.onToggleMultiSelection;
+				props.list.itemProps.onToggleAll = this.onToggleAllMultiSelection;
+				props.list.itemProps.isSelected = this.isSelected;
+				// selectedItems is part of the default state
+				// but host app can override it so
+				if (!state.selectedItems) {
```

Comment:
Is that really necessary ? This is a big constrains, that means that for every default state, we have to reset it each time it is set with an unwanted value.
I think we should not do that. Moreover, you set a new state within render, triggering a new render. If app overrides it, it should put a valid value.